ember-flight-icons/tests/dummy/app/routes/application.js

```diff
@@ -0,0 +1,27 @@
+import Route from '@ember/routing/route';
+import fetch from 'fetch';
+import { getOwner } from '@ember/application';
+
+export default class ApplicationRoute extends Route {
+  get contextRootURL() {
+    const config = getOwner(this).resolveRegistration('config:environment');
+    return config.rootURL || '/';
+  }
+
+  async model() {
+    // console.log('here', this.contextRootURL)
+
+    const response = await fetch(
+      `${this.contextRootURL}@hashicorp/ember-flight-icons/icons/_catalog.json`
```

Comment:
thought: how will we ensure that this file gets updated?

Reply:
yeah good one

(non-blocking) want to add, can you in a separate PR consider documenting the https://github.com/hashicorp/flight/tree/main/export-go/dist to https://github.com/hashicorp/flight/tree/main/ember-flight-icons/public/icons workflow, perhaps https://github.com/hashicorp/flight? 

alternatively... i think ideally we just want one source of truth. perhaps we can just use `export-go/dist` as `@hashicorp/flight-icons` ... then have `@hashicorp/flight-icons-ember` (to be renamed, if we agree) consume `@hashicorp/flight-icons`

i'm going to muddle with `@hashicorp/flight-icons` and make another version to npm, but i might try making the file from `export-go/dist` instead today

Reply:
right now it's pretty ad hoc (which is ok!) but we need to map out how exactly we want to connect the dots here, and I agree having a source of truth will be better than relying on manual copies. having the file live with the contents of `hashicorp/flight-icons` seems reasonable to me and is similar to what we do with structure-icons currently. For the purposes of documentation generation it _is_ possible to read the contents of the sprite itself and generate the list of icons based off of that, but that's sort of the same difference as reading from that file I suppose.